src/main/java/sample/Controller.java

```diff
@@ -1,5 +1,57 @@
 package sample;
 
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.TextField;
+import javafx.scene.control.PasswordField;
+import javafx.stage.Stage;
+
+
+import java.io.IOException;
+
 public class Controller {
 
-}
+    @FXML
+    private TextField username;
+    @FXML
+    private PasswordField password;
+
+    public void loginButtonIsClicked(ActionEvent event) throws IOException {
+        User user = new User(username.getText(), password.getText());
+        Database database = new Database();
+        List<User> userList = database.getUsers();
+        if(userExists(userList, user)){
```

Comment:
For the moment we are extracting all the data from the database and then we try to search for the certain user.
Can we optimize this a bit?